src/infra/workload/releaseunit/modules/stamp/keyvault-secrets.tf

```diff
@@ -16,6 +16,8 @@ locals {
     "CosmosDb-ApiKey"                            = data.azurerm_cosmosdb_account.global.primary_key
     "CosmosDb-DatabaseName"                      = var.cosmosdb_database_name
     "API-Key"                                    = var.api_key
+    "LogAnalytics-WorkspaceId"                   = data.azurerm_log_analytics_workspace.stamp.workspace_id
+    "MANAGED-IDENTITY-CLIENTID"                  = azurerm_kubernetes_cluster.stamp.kubelet_identity.0.client_id
```

Comment:
Is there any specific reason why we use uppercase for some and camelcase for others?

Reply:
nope :) just copy-and-paste... but I can switch all to camel

Reply:
fixed